### DISCUSSION TITLE
Assignment

### DISCUSSION THREAD

**Discussion by Aishbs** [microsoft/Web-Dev-For-Beginners#1115] [Q&A]
How should I answer the assignments? Should create a repo?

**Comment by smngvlkz** [comment]
I am also trying to figure this out. Were you able to find the answer?

**Reply by loki135** [reply]
hey if anyone found answer please let me know

**Reply by nikita539** [reply]
Hi, did you find an answer ?

**Reply by smngvlkz** [reply]
You follow the steps provided to you. Github Actions will detach if you have done everything right and it will automatically take you to the next page/step after you have saved everything

**Reply by nikita539** [reply]
How does Github Actions check my my code, and where i should write my code for correct work of Github Actions  ?

**Comment by mahema-14** [comment]
Hey! Here's what I'd do:

1) Create a new repo named after the assignment
2) Add your code and files
3) Write a small README explaining what it's about
4) Submit the repo link  and that's it...

Keeps everything clean and easy to review. If your course has specific instructions though, follow those first!
